rex-release/out.sh

```diff
@@ -47,6 +47,20 @@ upload-release() {
 
   # books files with explicit content type set because they don't have extensions, loaded unversioned so no cloudfront caching
   aws s3 sync --exclude 'service-worker.js' --content-type 'text/html' --cache-control 'max-age=0' "$path/books/" "s3://$bucket/rex/releases/$version/books"
+
+  # configure redirects
+  for row in $(cat $path/rex/redirects.json | jq -r '.[] | @base64'); do
+    getField() {
+      echo ${row} | base64 --decode | jq -r ${1}
+    }
+
+    from=$(echo $(getField '.from'))
+    to=$(echo $(getField '.to'))
+
+    if [ ! -e "$path/${from}" ] && [ -e "$path/${to}" ]; then
+      aws s3api put-object --bucket $from --key $from-key --website-redirect-location $to
+    fi
```

Comment:
i feel like we should do something in an else here... probably just log something. we'll have to add some validation that actually fails earlier in the process. at this point if we bailed that might be worse

(we'll do a separate task for more validation)